axlearn/audio/asr_encoder_test.py

```diff
@@ -243,6 +248,8 @@ def test_asr_encoder(self, is_training: bool):
         self.assertEqual(output_paddings.shape, (batch_size, output_shape[1]))
         self.assertTrue(jnp.all(output_paddings[:2] == output_paddings[2:]))
 
-        # If is_training, outputs should always be different due to augmentation.
+        # If is_training and use_augmenter, outputs should always be different due to augmentation.
         # Otherwise, outputs should be the same despite differences in padding.
-        self.assertEqual(not is_training, jnp.allclose(outputs[:2], outputs[2:]))
+        self.assertEqual(
+            not (is_training and use_augmenter), jnp.allclose(outputs[:2], outputs[2:])
```

Comment:
nit: I am a little confused about why `outputs[:2]` should be the same as `outputs[2:]`, could you add a comment to explain why they are the same in this case ?

Reply:
Done.